docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -276,6 +276,16 @@ The following command retrieves all rule groups in the Grafana Mimir instance an
 mimirtool rules print
 ```
 
+To save all the rules for editing and re-upload to Mimir, use the `--output-dir` flag.
```

Comment:
nit: I think that it's useful to distinguish between options and flags. Where a flag is an option that is a boolean toggle. In this case, and with that distinction, this is an option but not a flag.

```suggestion
To save all the rules for editing and re-upload to Mimir, use the `--output-dir` option.
```